config/locales/en/course/forum/topics.yml

```diff
@@ -17,3 +17,27 @@ en:
         destroy:
           success: 'Topic %{title} was deleted.'
           failure: 'Failed to delete topic: %{error}'
+        subscribe:
+          tag: 'Subscribe'
+          success: 'You have subscribed to Topic %{title}.'
```

Comment:
the rest of the messages do not have it right? It would be troublesome because we need to parse the title for html and sanitize also